.github/workflows/bump.yml

```diff
@@ -0,0 +1,113 @@
+name: Bump Version
+
+on:
+  repository_dispatch:
+    types:
+    - zitadel-released
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  bump:
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+    - id: checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Read Current Chart Version
+      id: current-chart-version
+      uses: jbutcher5/read-yaml@1.6
+      with:
+        file: './charts/zitadel/Chart.yaml'
+        key-path: '["version"]'
+
+    - name: Read Current ZITADEL Version
+      id: current-zitadel-version
+      uses: jbutcher5/read-yaml@1.6
+      with:
+        file: './charts/zitadel/Chart.yaml'
+        key-path: '["appVersion"]'
+
+    - name: Print Current Versions
+      run: |
+        echo "Chart Version: ${{ steps.current-chart-version.outputs.data }}"
+        echo "ZITADEL Version: ${{ steps.current-zitadel-version.outputs.data }}"
+
+    - name: Parse Last ZITADEL Version
+      id: parsed-last-zitadel-version
+      uses: booxmedialtd/ws-action-parse-semver@v1
+      with:
+        input_string: ${{ steps.current-zitadel-version.outputs.data }}
+
+    - name: Set Version Type
```

Comment:
can this not be handled by semantic? this seems a bit complicated and manual

Reply:
the semantic release action still doesn't output that information, so we would have to hack eighter way  https://github.com/marketplace/actions/action-for-semantic-release#outputs

Resolving this conversation